lib/mindwendel_web/live/idea_live/index_component.html.leex

```diff
@@ -2,9 +2,17 @@
   <div id="ideas" class="row mb-5">
     <%= for idea <- @ideas do %>
       <div class="col-md-6">
-        <div class="card mt-3 shadow-sm p-2 rounded IndexComponent__IdeaCard--labelled-<%= idea.label %>" id="idea-<%= idea.id %>">
+        <%= unless idea.label do %>
+          <div class="card mt-3 shadow-sm p-2 rounded IndexComponent__IdeaCard" data-testid="<%= idea.id %>">
```

Comment:
data-testId? is the name correct?

Reply:
I followed the default naming proposed by the react testing library, see https://testing-library.com/docs/queries/bytestid  or https://reactjs.org/docs/testing-recipes.html. This lib and other react example use `data-testid`.

I am open for suggestions.